### ISSUE TITLE
Vulnerability found in diff in package-lock.json

### ISSUE THREAD

**Issue by schubon** [IBMStreams/vscode-ide#48]
### Details

WS-2018-0590
high severity
Vulnerable versions: < 3.5.0
Patched version: 3.5.0

A vulnerability was found in diff before v3.5.0, the affected versions of this package are vulnerable to Regular Expression Denial of Service (ReDoS) attacks.

### Remediation

Upgrade diff to version 3.5.0 or later. For example:

```
"dependencies": {
  "diff": ">=3.5.0"
}
```
or…

```
"devDependencies": {
  "diff": ">=3.5.0"
}
```

Always verify the validity and compatibility of suggestions with your codebase.

**Comment by queeniema** [comment]
The issue seems to be stemming from the `vscode` NPM package so I opened an issue in the vscode repo: https://github.com/microsoft/vscode/issues/75732.

**Comment by queeniema** [comment]
This issue was fixed in version 1.1.35 of `vscode`. Updating our dependency to use the latest version.

**Comment by queeniema** [comment]
<img width="432" alt="Screen Shot 2019-07-01 at 1 43 19 PM" src="https://user-images.githubusercontent.com/2454818/60465576-47367980-9c06-11e9-89eb-d62eeab9eaf6.png">
